src/main/java/ToDo.java

```diff
@@ -0,0 +1,9 @@
+/**
+ * This class represents ToDo tasks which is a type of Task.
+ */
+public class ToDo extends Task {
+    public ToDo(String description) {
+        super(description);
+        this.typeOfTask = "[T]";
```

Comment:
Not really a suggestion again, just curious why there's a need for a typeOfTask variable when you already have the different types of tasks as individual classes inheriting the Task class!